resources.html

```diff
@@ -82,10 +82,6 @@
           <div class="intro-heading">Chaincode Residency 2020</div>
           <div class="intro-lead-in">Lightning: June 1st - 5th</div>
```

Comment:
Probably worth pulling the dates from here as well.

Reply:
done